dsgrid/config/project_config.py

```diff
@@ -384,6 +380,26 @@ def get_base_dimension(self, dimension_type: DimensionType):
                 return dim_config
         assert False, dimension_type
 
+    def get_dimension_records(self, dimension_type: DimensionType, query_name: str):
```

Comment:
Why do we need dimension type here if query_name is unique within a project (including dataset and dimension)?